src/vma/proto/mem_buf_desc.h

```diff
@@ -128,6 +134,9 @@ class mem_buf_desc_t {
 			};
 
 			void* 		context;
+#ifdef DEFINED_VMAPOLL 
+			bool 		vma_polled;
+#endif // DEFINED_VMAPOLL 			
```

Comment:
`vma_polled` is vma_poll specific variable. It means nothing for legacy mode. So it can be visible for all configurations and used in different parts of code to differ modes w/o using a lot of `DEFINED_VMAPOLL`
Please consider enable one for legacy and vma_poll and use this to reduce `DEFINED_VMAPOLL`